### PR TITLE
feat : add mobile navigation with hamburger menu

### PR DESCRIPTION
**Pull request by viru2001**
##  add mobile navigation with hamburger menu

image

image

[Deploy Preview](https://deploy-preview-16--cool-ui.netlify.app/pages/docs.html#navigation)

### CODE REVIEW

**Review by yogesh- — APPROVED** [review]
LGTM

File: components/Navigation/navigation.css [hunk 1]
```diff
@@ -25,3 +25,51 @@
...
+
+/* navbar */
+.brand-name {
+  color: var(--primary-color);
+}
+.text-dec-none {
+  text-decoration: none;
+}
+.search-box {
+  width: 50rem;
```
Comment (mohitdhatrak) [inline components/Navigation/navigation.css]: i think its better to have 1 line space between CSS blocks for better readability

**Comment by shlok30** [comment]
Looks good to merge

File: components/Navigation/navigation.css [hunk 2]
```diff
@@ -25,3 +25,51 @@
...
+}
+.search-input {
+  width: 45rem;
+}
+.nav-link-icon {
+  cursor: pointer;
```
Comment (mohitdhatrak) [inline components/Navigation/navigation.css]: if it's a link it doesnt require cursor pointer, links have it by default

**Review by shlok30 — APPROVED** [review]
Looks good to merge

**Review by mohitdhatrak — COMMENTED** [review]
In mobile view, the hamburger one is going a bit out of screen, you can fix that. Submission first though!

File: components/Navigation/navigation.css (hunk 1 above)
Comment (viru2001) [reply components/Navigation/navigation.css]: ok will do it

File: components/Navigation/navigation.css (hunk 2 above)
Comment (viru2001) [reply components/Navigation/navigation.css]: yes it's a link , i will remove this

**Comment by viru2001** [comment]
> In mobile view, the hamburger one is going a bit out of screen, you can fix that. Submission first though!

for me, it's working fine. The documentation page is not responsive, so maybe that's why its going out of screen for you

**Pull request merged**